ansible/envs/dev.yml

```diff
@@ -4,3 +4,4 @@ telemetry_analysis_spark_emr_bucket: telemetry-spark-emr-2
 telemetry_analysis_public_bucket: telemetry-public-analysis-2
 telemetry_analysis_private_bucket: telemetry-private-analysis-2
 telemetry_analysis_anaconda_path: /mnt/anaconda2
+telemetry_analysis_metastore_stack_name: telemetry-metastore
```

Comment:
Notably this differs from https://github.com/mozilla/emr-bootstrap-presto/blob/master/envs/default.yml#L22 but appears to better reflect reality.